packages/manager/src/factories/databases.ts

```
@@ -238,11 +238,7 @@ export const databaseFactory = Factory.Sync.makeFactory<Database>({
   },
   oldest_restore_time: '2024-09-15T17:15:12',
   platform: Factory.each((i) => (adb10(i) ? 'rdbms-legacy' : 'rdbms-default')),
-  private_network: {
-    public_access: false,
-    subnet_id: 1,
-    vpc_id: 123,
-  },
+  private_network: null,
```

This is to clean up the mock data for databases from my [previous VPC PR](https://github.com/linode/manager/pull/12324). This was used to test VPC selection, but was leftover in the previous PR and is no longer needed. It can fail find the VPC from the mock VPC list data so I chose to clean it up this PR. This fixes that issue by instead mocking a database with no VPC configured.